components/SubmissionsTable/SubmissionRow.tsx

```diff
@@ -0,0 +1,135 @@
+import { Submission } from 'data/flexibleForms/forms.types';
+import Link from 'next/link';
+import { format } from 'date-fns';
+import forms from 'data/flexibleForms';
+import { User } from 'types';
+import s from './index.module.scss';
+
+interface Props {
+  submission: Submission;
+  openRow: string | false;
+  setOpenRow: (value: string | false) => void;
+  user: User;
+}
+
+const SubmissionRow = ({
+  submission,
+  openRow,
+  setOpenRow,
+  user,
+}: Props): React.ReactElement => {
+  const open = openRow === submission.submissionId;
+
+  const form = forms.find((form) => form.id === submission.formId);
```

Comment:
Do we want to do an explicit check here for `form === false` and return null? I can see a scenario where, for whatever reason, in the future a submission is available for a form that has been renamed or no longer exists, and I think this would blow up.

Something like:
```suggestion
  const form = forms.find((form) => form.id === submission.formId);
  
  if(!form) {
    return null;
  }
```